constants.js

```diff
@@ -86,6 +86,16 @@ if (!abilities.nevermore_shadowraze3)
   // Find and replace short raze range with long raze range
   abilities.nevermore_shadowraze3.attrib = abilities.nevermore_shadowraze3.attrib.replace(/\d{3}/, 700);
 }
+if (!abilities.beastmaster_call_of_the_wild_boar)
+{
+  abilities.beastmaster_call_of_the_wild_boar = Object.assign(
```

Comment:
[https://github.com/dotabuff/d2vpkr/blob/9631d01d5bab8b4555432a434f0199a2b685f9cd/dota/resource/dota_english.json](https://github.com/dotabuff/d2vpkr/blob/9631d01d5bab8b4555432a434f0199a2b685f9cd/dota/resource/dota_english.json)

`      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar": "Call of the Wild: Boar",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_Description": "Beastmaster calls a powerful Boar to stalk the battlefield, capable of spitting poison at enemies to slow their movement and attack speed.",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_Lore": "After befriending the strange beast of his childhood, Beastmaster has learned to call to animals in times of need.",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_boar_hp_tooltip": "BOAR HEALTH:",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_boar_damage_tooltip": "BOAR ATTACK DAMAGE:",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_boar_moveslow_tooltip": "%BOAR MOVEMENT SLOW:",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_poison_duration_tooltip": "BOAR POISON DURATION:",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_boar_attackslow_tooltip": "BOAR ATTACK SLOW:",
      "DOTA_Tooltip_ability_beastmaster_call_of_the_wild_boar_boar_duration": "BOAR DURATION:",`